### DISCUSSION TITLE
Won't have any reaction after clicking Reboot Now when creating ubuntu vm - mac m1

### DISCUSSION THREAD

**Discussion by liubq7** [utmapp/UTM#3931] [UTM Help (macOS)]
**Describe the issue**
I'd like to create an ubuntu 20.04 vm. After the installation completed, I clicked `Reboot Now`. but it just keeps loading like forever.
<img width="770" alt="截屏2022-04-14 13 29 37" src="https://user-images.githubusercontent.com/55693386/163319818-1c9f891f-74c1-4c2e-add3-f1ffe113dde0.png">

<img width="322" alt="截屏2022-04-14 13 09 55" src="https://user-images.githubusercontent.com/55693386/163318721-030fd137-bd3b-4fe4-bede-4d891958ba66.png">

**Configuration**
* UTM Version: 3.1.5 (53)
* OS Version: Monterey 12.3.1
* Apple Silicon

**Debug log**
[debug.log](https://github.com/utmapp/UTM/files/8486475/debug.log)

Running:  -L /Applications/UTM.app/Contents/Resources/qemu -S -qmp tcp:127.0.0.1:4000,server,nowait -nodefaults -vga none -spice "unix=on,addr=/Users/liubeiqian/Library/Group Containers/WDNLXAD4W8.com.utmapp.UTM/1484D9FF-B0FC-4BF5-B846-74C5B57B58DE.spice,disable-ticketing=on,image-compression=off,playback-compression=off,streaming-video=off,gl=off" -device virtio-ramfb -device virtio-rng-pci -cpu host -smp cpus=8,sockets=1,cores=8,threads=1 -machine virt,highmem=off -accel hvf -accel tcg,tb-size=1024 -drive if=pflash,format=raw,unit=0,file=/Applications/UTM.app/Contents/Resources/qemu/edk2-aarch64-code.fd,readonly=on -drive if=pflash,unit=1,file=/Users/liubeiqian/Library/Containers/com.utmapp.UTM/Data/Documents/Linux.utm/Images/efi_vars.fd -boot menu=on -m 4096 -device intel-hda -device hda-duplex -name Linux -device nec-usb-xhci,id=usb-bus -device usb-tablet,bus=usb-bus.0 -device usb-mouse,bus=usb-bus.0 -device usb-kbd,bus=usb-bus.0 -device ich9-usb-ehci1,id=usb-controller-0 -device ich9-usb-uhci1,masterbus=usb-controller-0.0,firstport=0,multifunction=on -device ich9-usb-uhci2,masterbus=usb-controller-0.0,firstport=2,multifunction=on -device ich9-usb-uhci3,masterbus=usb-controller-0.0,firstport=4,multifunction=on -chardev spicevmc,name=usbredir,id=usbredirchardev0 -device usb-redir,chardev=usbredirchardev0,id=usbredirdev0,bus=usb-controller-0.0 -chardev spicevmc,name=usbredir,id=usbredirchardev1 -device usb-redir,chardev=usbredirchardev1,id=usbredirdev1,bus=usb-controller-0.0 -chardev spicevmc,name=usbredir,id=usbredirchardev2 -device usb-redir,chardev=usbredirchardev2,id=usbredirdev2,bus=usb-controller-0.0 -device usb-storage,drive=cdrom0,removable=true,bootindex=0,bus=usb-bus.0 -drive if=none,media=cdrom,id=cdrom0 -device virtio-blk-pci,drive=drive0,bootindex=1 -drive if=none,media=disk,id=drive0,file=/Users/liubeiqian/Library/Containers/com.utmapp.UTM/Data/Documents/Linux.utm/Images/data.qcow2,cache=writethrough -device virtio-net-pci,mac=7E:89:17:D6:DE:3F,netdev=net0 -netdev vmnet-macos,mode=shared,id=net0 -device virtio-serial -device virtserialport,chardev=vdagent,name=com.redhat.spice.0 -chardev spicevmc,id=vdagent,debug=0,name=vdagent -device virtserialport,chardev=charchannel1,id=channel1,name=org.spice-space.webdav.0 -chardev spiceport,name=org.spice-space.webdav.0,id=charchannel1 -uuid 1484D9FF-B0FC-4BF5-B846-74C5B57B58DE -rtc base=localtime
qemu-aarch64-softmmu: -netdev vmnet-macos,mode=shared,id=net0: info: Started vmnet interface with configuration:
qemu-aarch64-softmmu: -netdev vmnet-macos,mode=shared,id=net0: info: MTU:              1500
qemu-aarch64-softmmu: -netdev vmnet-macos,mode=shared,id=net0: info: Max packet size:  1514
qemu-aarch64-softmmu: -netdev vmnet-macos,mode=shared,id=net0: info: MAC:              a2:c0:69:9f:d0:46
qemu-aarch64-softmmu: -netdev vmnet-macos,mode=shared,id=net0: info: DHCP IPv4 start:  192.168.64.1
qemu-aarch64-softmmu: -netdev vmnet-macos,mode=shared,id=net0: info: DHCP IPv4 end:    192.168.64.254
qemu-aarch64-softmmu: -netdev vmnet-macos,mode=shared,id=net0: info: IPv4 subnet mask: 255.255.255.0
qemu-aarch64-softmmu: -netdev vmnet-macos,mode=shared,id=net0: info: UUID:             CE367E0E-A7F5-4709-8292-7CCD77F2FABA
qemu-aarch64-softmmu: warning: Spice: playback:0 (0x147811140): setsockopt failed, Operation not supported on socket
qemu-aarch64-softmmu: warning: Spice: record:0 (0x1478111f0): setsockopt failed, Operation not supported on socket

**Comment by osy** [comment]
This is an unfortunate bug in QEMU. The guide tells you to just ignore it and manually restart https://mac.getutm.app/gallery/ubuntu-20-04

**Reply by tyliec** [reply]
Running into the same problem on:
- MacOS Ventura 13.4.1 (22F82)
- UTM Version 4.2.5 (81)
- Apple M1 Max
